tests/tck/features/fetch/FetchEdges.intVid.feature

```diff
@@ -22,17 +22,17 @@ Feature: Fetch Int Vid Edges
     # Fetch prop on an edge without yield
     When executing query:
       """
-      FETCH PROP ON serve hash("Boris Diaw")->hash("Spurs")
+      FETCH PROP ON serve hash("Boris Diaw")->hash("Spurs") YIELD edge
```

Comment:
Don't modify the origin case if it's compatible.

Reply:
will fix it